src/ServiceControl.Monitoring/Hosting/Commands/RunCommand.cs

```diff
@@ -52,7 +52,7 @@ class OnConsoleCancel
         {
             Action action;
 
-            private OnConsoleCancel(Action action)
+            OnConsoleCancel(Action action)
```

Comment:
This is my `TeamCity` kungfu failing me when checking for inspection errors. I can undo it if needed.